README.md

```diff
@@ -81,12 +81,13 @@ To build the website locally, follow these steps:
   - If you don't have access to Ruby 3.2 or later in your distribution's repositories, install a compatible version of Ruby using `rbenv`:
     - Install in Ubuntu:
       - `sudo apt install rbenv`
+      - `sudo apt install libyaml-dev`
```

Comment:
This is required for rbenv to build ruby 3.2.5. I'm sure there are other dependencies though. So we should consider enumerating all of the dependencies somehow instead of relying on contributors just happening to have the required dependencies. For now, this is an improvement though